internal/webhook/module.go

```diff
@@ -212,12 +212,16 @@ func validateModprobe(modprobe kmmv1beta1.ModprobeSpec) error {
 
 	return nil
 }
-func validateModuleTolerarations(mod *kmmv1beta1.Module) error {
+func validateModuleTolerations(mod *kmmv1beta1.Module) error {
 	for _, toleration := range mod.Spec.Tolerations {
-		switch toleration.Effect {
-		case corev1.TaintEffectNoSchedule, corev1.TaintEffectNoExecute, corev1.TaintEffectPreferNoSchedule:
-			continue
-		default:
+
+		if toleration.Operator != corev1.TolerationOpExists && toleration.Operator != corev1.TolerationOpEqual {
+			return fmt.Errorf("invalid toleration operator %s. allowed values are Exists, Equal", toleration.Operator)
```

Comment:
Can we try to find an error message without a `.` in it? When we see a full error in the code that has be wrapped multiple time it is easier when each error message is a single, short error message.